bindings/ergo-lib-ios/Tests/ErgoLibTests/TokenTests.swift

```diff
@@ -44,14 +44,16 @@ final class TokenIdTests: XCTestCase {
         let amount = Int64(12345678)
         let tokenAmount = try TokenAmount(fromInt64: amount)
         let token = Token(tokenId: tokenId, tokenAmount: tokenAmount)
+
+        let maxTokensCount = UInt(100)
         
-        for _ in 1...255 {
+        for _ in 1...maxTokensCount {
             try tokens.add(token: token)
         }
-        XCTAssertEqual(tokens.len(), UInt(255))
-        XCTAssertNotNil(tokens.get(index: 254))
+        XCTAssertEqual(tokens.len(), UInt(maxTokensCount))
```

Comment:
I noticed that we don't have associated tests of this sort for WASM. Probably a good idea to have at least this test included. If you like I can make an issue for it, and I will get around to implementing it.

Reply:
Good catch! Actually, I think `Tokens` type in both C and Wasm layers should not limit how many tokens they hold via `add_token()`. The max token limit exists only in the `ErgoBox` and should be enforced only there. For example, `Tokens` type is used in `BoxSelector` to represent target tokens. There is no limit there. One can select as many tokens as they want, given that they will be placed later into separate output boxes according to the max token limit per box. Which is enforced in `ErgoBoxCandidateBuilder::build()` and in `ErgoBox` itself via `BoundedVec`.
We have https://github.com/ergoplatform/sigma-rust/issues/594 where we can also remove the `Tokens::add_token()` check.

Reply:
Thanks, I got it.